crate/kmip/src/crypto/cover_crypt/kmip_requests.rs

```diff
@@ -60,6 +61,7 @@ pub fn build_create_user_decryption_private_key_request<T: IntoIterator<Item = i
                 cover_crypt_master_private_key_id.to_owned(),
             ),
         }]),
+        cryptographic_usage_mask: Some(CryptographicUsageMask::Unrestricted),
```

Comment:
Because we discussed with @bgrieder and concluded that Covercrypt keys could be defaulted to `Unrestricted` mostly because they are not used in FIPS mode